docs/pages/omnia3/Modeler/omnia3_modeler_queries.md

```diff
@@ -0,0 +1,204 @@
+    FROM vw_MyDocument document
+    JOIN vw_MyDocument_Lines lines on lines.identifier = document.identifier
+    GROUP BY document._code, lines._resource
+```
+
+### _Test changes_
+
+It's possible to test changes to an advanced query by clicking the _Execute_ button.
+
+The test is only available to queries that execute on the local _System_ datasource, and it has two possible results: Error or Success.
+
+When a test executes successfully, the following information is returned:
+
+- Execution Time: The time the query takes to be executed on the database, in milliseconds
+- Total response time: The total execution time (in milliseconds), from the moment when the test request is sent to Omnia API until the response is received
+- Data: The first 50 records returned by the query execution
+
+When a test executes with error, the following information is returned:
+
+- Message: The error message returned by the query execution
+- Hints: Hints to fix the error returned by the database server. E.g. a typo on a column name
+
+_NOTE: Since the query executes on the tenant database schema, it's important to have a model built with all changes that have an impact on the database structure. Important changes include the addition of new Entities and new Attributes_
+
+### _Views Tree_
+
+On the right of the _SQL Query_ there's a sidebar containing all the compiled **view's** of the model and respective **columns** and **types**. You can also **type search** for a view and column identifier.
+
+<p align="center">
+  <img src="/images/modeler/ViewsTree.png">
+</p>
+
+It's possible to **drag** (hold the mouse click on) each _view_ or _column's_ _name_ and **drop** it (release the mouse click) on the _SQL Query_ on left. There's also an option to generate a `SELECT` script for each _view_. Check each example below:
+
+- When **dropping** a _view_ of an entity named _company_ the following information is returned:
+
+```SQL
+    vw_company
+```
+
+- When **dropping** a _column_ of an attribute named _sales_ from an entity named _company_ the following information is returned:
+  - Notice the `,` in the end of the _column_ drop result. We've added it to ease the sequential addition of _columns_.
+
+```SQL
+    vw_company.sales,
+```
+
+- When **dropping** the generated `SELECT` script a view _vw_company_ with a column _sales_ and the 3 other **default** _columns_ (`identifier`, `_create_at`, `_updated_at`) the following information is returned:
+
+```SQL
+    SELECT
+        vw_company.identifier,
+        vw_company._created_at,
+        vw_company._updated_at,
+        vw_company.sales
+    FROM vw_company
+```
+
+### _Output Structure_
+
+On Advanced Queries, it's possible to define the output structure. This structure can be used later when creating a new list, to define the columns.
+
+To set the query output structure, on the advanced query editor navigate to Columns tab and click on the _Auto-fill_ button:
```

Comment:
Auto-fill only in system

Reply:
Describe that even though auto-fill is only available in system, queries should always be typed